Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -297,4 +228,96 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
         }
         """
     }
+
+    private static func publicBundleAccessorString(for target: Target) -> String {
+        """
+        // MARK: - Objective-C Bundle Accessor
+        @objc
+        public class \(target.productName.toValidSwiftIdentifier())Resources: NSObject {
+        @objc public class var bundle: Bundle {
+            return .module
+        }
+        }
+        """
+    }
+
+    private static func swiftSPMBundleAccessorString(for target: Target, and bundleName: String) -> String {
+        """
+        // MARK: - Swift Bundle Accessor - for SPM
+        private class BundleFinder {}
+        extension Foundation.Bundle {
+        /// Since \(target.name) is a \(
+            target
+                .product
+        ), the bundle containing the resources is copied into the final product.
```

Comment:
This formatting is slightly odd. Is it actually treated as one comment? 🤔

Reply:
I didn't touch this formatting, mise lint didn't change it, but indeed its considered as one comment - i.e one line

Example from the `app_with_spm_dependencies`, the derived  file `Derived/Sources/TuistBundle+Styles`

```swift
...
/// Since Styles is a staticFramework, the bundle containing the resources is copied into the final product.
...
```

Reply:
I can change it if you wish